src/daemons/MetaDaemonInit.cpp

```diff
@@ -87,13 +87,13 @@ std::unique_ptr<nebula::kvstore::KVStore> initKV(std::vector<nebula::HostAddr> p
   auto kvstore = std::make_unique<nebula::kvstore::NebulaStore>(
       std::move(options), ioPool, localhost, threadManager);
   if (!(kvstore->init())) {
-    LOG(ERROR) << "Nebula store init failed";
+    LOG(ERROR) << "NebulaGraph store init failed";
```

Comment:
Better don't modify it or use `Nebula store` or `NebulaStore`, it is a internal component